microscopic-monks/primal/resources/items.json

```diff
@@ -0,0 +1,12 @@
+{
+  "rock": {"health": 0, "range": 0, "recipe": {}, "source": "r.png", "damage": 0},
```

Comment:
r.png sounds like something that should be named rock.png. the use of non-descriptive names is a bit troublesome.